.travis.yml

```diff
@@ -0,0 +1,23 @@
+
+sudo: required 
+dist: trusty 
```

Comment:
Travis does not support `xenial` atm and isn't planning to for more than 6 months. Currently `industrial_ci` runs kinetic items in a Docker vm.